include/bluetooth/mesh/light_xyl_cli.h

```diff
@@ -0,0 +1,324 @@
+					struct bt_mesh_msg_ctx *ctx,
+					const struct bt_mesh_light_xyl *set);
+
+/** @brief Get the Light xyL state of the bound server.
+ *
+ * This call is blocking if the @c rsp buffer is non-NULL. Otherwise, this
+ * function will return.
+ *
+ * @param[in] cli Client model to send on.
+ * @param[in] ctx Message context, or NULL to use the configured publish
+ * parameters.
+ * @param[in] rsp Status response buffer, or NULL to keep from blocking.
+ *
+ * @retval 0 Successfully sent the message and populated the @c rsp buffer.
+ * @retval -EALREADY A blocking request is already in progress.
+ * @retval -EADDRNOTAVAIL A message context was not provided and publishing is
+ * not configured.
+ * @retval -EAGAIN The device has not been provisioned.
+ * @retval -ETIMEDOUT The request timed out without a response.
+ */
+int bt_mesh_light_xyl_range_get(struct bt_mesh_light_xyl_cli *cli,
+				struct bt_mesh_msg_ctx *ctx,
+				struct bt_mesh_light_xyl_range_status *rsp);
+
+/** @brief Set the xyL Range of the server.
+ *
+ * This call is blocking if the @c rsp buffer is non-NULL. Otherwise, this
+ * function will return.
+ *
+ * @param[in] cli Client model to send on.
+ * @param[in] ctx Message context, or NULL to use the configured publish
+ * parameters.
+ * @param[in] set Range to set.
+ * @param[in] rsp Response status buffer, or NULL to keep from blocking.
+ *
+ * @retval 0 Successfully sent the message and populated the @c rsp buffer.
+ * @retval -EALREADY A blocking request is already in progress.
+ * @retval -EADDRNOTAVAIL A message context was not provided and publishing is
+ * not configured.
+ * @retval -EAGAIN The device has not been provisioned.
+ * @retval -ETIMEDOUT The request timed out without a response.
+ * @retval -EFAULT Invalid input range.
+ */
+
+int bt_mesh_light_xyl_range_set(struct bt_mesh_light_xyl_cli *cli,
+				struct bt_mesh_msg_ctx *ctx,
+				const struct bt_mesh_light_xy_range *set,
+				struct bt_mesh_light_xyl_range_status *rsp);
+
+/** @brief Set the xyL Range of the server without requesting a
+ * response.
+ *
+ * @param[in] cli Client model to send on.
+ * @param[in] ctx Message context, or NULL to use the configured publish
+ * parameters.
+ * @param[in] set Range state to set.
+ *
+ * @retval 0 Successfully sent the message.
+ * @retval -EADDRNOTAVAIL A message context was not provided and publishing is
+ * not configured.
+ * @retval -EAGAIN The device has not been provisioned.
+ * @retval -EFAULT Invalid input range.
+ */
+int bt_mesh_light_xyl_range_set_unack(struct bt_mesh_light_xyl_cli *cli,
+				      struct bt_mesh_msg_ctx *ctx,
+				      const struct bt_mesh_light_xy_range *set);
+
+/** @cond INTERNAL_HIDDEN */
+extern const struct bt_mesh_model_op _bt_mesh_light_xyl_cli_op[];
+extern const struct bt_mesh_model_cb _bt_mesh_light_xyl_cli_cb;
+/** @endcond */
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif /* BT_MESH_LIGHT_XYL_CLI_H__ */
+
+/* @} */
```

Comment:
This needs to start with `/**` to resolve the doc error. Check for other headers as well.